README.md

```diff
@@ -1,15 +1,565 @@
-# FI-WARE Connectors
+# Cygnus connector
```

Comment:
Regarding root README, what I have done is to delete the old one and move the older flume/README.md to the root directory. I think the only think we "lost" is the reference to resources/ directory, but I think it doesn't mind.